Containers: use untested or official container images accordingly (#12698)
os-autoinst/os-autoinst-distri-opensuse: variables.md

CHECK_RELEASENOTES_ORIGIN | boolean | false | Loads `installation/releasenotes_origin` test module.
CHECKSUM_* | string | | SHA256 checksum of the * medium. E.g. CHECKSUM_ISO_1 for ISO_1.
CHECKSUM_FAILED | string | | Variable is set if checksum of installation medium fails to visualize error in the test module and not just put this information in the autoinst log file.

> CONTAINERS_UNTESTED_IMAGES | boolean | false | Used by container tests to decide if pulling an unreleased image from registry.suse.de or a released image from registry.suse.com.

Reviewer: SLE specific.
```suggestion
CONTAINERS_UNTESTED_IMAGES | boolean | false | Whether to use `untested_images` or `official_images` from `lib/containers/urls.pm`.
```
